extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -386,3 +386,15 @@ export async function getFileShares(account: azdata.Account, subscription: azure
 		errors: response.errors ? response.errors : []
 	};
 }
+
+export async function getMigrationControllers(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, resourceGroupName: string, regionName: string, ignoreErrors: boolean): Promise<GetMigrationControllersResult> {
+	const apiEndpoint = `https://` + regionName + `.management.azure.com` +
```

Comment:
I've not seen the region name prefixed to the host like this before - is this documented anywhere on why it's being done this way?

Reply:
These are canary apis and therefore are available only on a particular region. This also explains the different version number

Reply:
A comment explaining this would be nice.

Is the intention that this is temporary then and when the feature is GA'd the URI would be updated to be the same as all the others?

Reply:
Yup, this will be used for the preview and the GAed apis will replace it.